modules/ROOT/pages/6.3-release-notes.adoc

```diff
@@ -0,0 +1,692 @@
+
+The basic setup for this new option as follows.
+
+[source,js]
+----
+tinymce.init({
+  selector: "textarea", // change this value according to your HTML
+  plugins: "advcode",
+  advcode_inline: true, // To use this option, set this value to "true".
+  toolbar: "code",
+});
+----
+
+=== Checklist 2.0.2
+
+The {productname} 6.3 release includes an accompanying release of the **Checklist** premium plugin.
+
+**Checklist** 2.0.2 includes one fix:
+
+==== Checklist toggling did not preserve selection when used on an existing list
+
+In previous {productname} versions, when selecting content over either an ordered or unordered list, then switching the list to a checklist, the user’s content selection was not preserved.
+
+In {productname} 6.3, the user selection is now preserved when toggling a list to a checklist.
+
+For information on the **Checklist** plugin, see xref:checklist.adoc[Checklist].
+
+=== Comments 3.2.0
+
+The {productname} 6.3 release includes an accompanying release of the **Comments** premium plugin.
+
+**Comments** 3.2.0 includes one fix and one change:
+
+==== A comment could be inserted when in an empty editor or block
+
+In previous {productname} versions a bug existed that allowed comments to be placed into an empty editor or block.
+
+With this update, the comments button, menu-item, comment sidebar and shortcut keys are now disabled when the editor contains no content.
+
+==== The selection now expands to encompass the word containing the insertion point when a comment is added
+
+{productname} 6.3 adds xref:additions[two new API functions] — `tinymce.selection` and `tinymce.dom.RangeUtils`.
+
+With these two functions available, if there is no selected text when a new comment is added, {productname} automatically extends the selection to encompass the word containing the insertion point and links the newly added comment to this automatically selected word.
+
+For information on the **Comments** plugin, see: xref:introduction-to-tiny-comments.adoc[Introduction to Tiny Comments].
+
+
+=== Merge Tags 1.1.0
+
+The {productname} 6.3 release includes an accompanying release of the **Merge Tags** premium plugin.
+
+**Merge Tags** 1.1.0 includes one addition, one fix, and one change:
+
+==== New mceMergeTag command to wrap selected content in a merge tag
+
+A new command, `mceMergeTag` has been added which can convert any string into a merge tag. This new command is available to use when defining xref:content-behavior-options.adoc#text_patterns[Text Patterns].
+
+Previously only a predefined list of merge tags could be used. When the new `mceMergeTag` command is added, any valid Merge Tag string can be converted into a merge tag.
+
+As well, configuring the inline `text_pattern` with this command allows merge tags to be entered by manual typing.
+
+Type the prefix, some content, the suffix, and then a word boundary character, and the `mceMergeTag` command will immediately wrap that content in a complete merge tag.
```

Comment:
How about references to the usage of `text_patterns` option here.

Reply:
That’s what the paragraph above does:

1. configuring the inline `text_pattern` with this command [ie, `mceMergeTags`] allows…
2. example of what such configuring allows.